ceph-nvmeof.conf

```diff
@@ -37,7 +37,8 @@ client_cert = ./client.crt
 
 [spdk]
 tgt_path = /usr/local/bin/nvmf_tgt
-rpc_socket = /var/tmp/spdk.sock
+#rpc_socket_dir = /var/tmp/
```

Comment:
Why are these two lines commented?

Reply:
@barakda I left the lines commented so the use would know these attributes exist in case he would like to set them. This is done elsewhere in the conf file, like in:
```
#tgt_cmd_extra_args = --env-context="--no-huge -m1024" --iova-mode=va
```